h/sentry/helpers/event.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""
+Decorator class for Sentry events.
+
+Centralizes Sentry event parsing in one reusable place. Sentry events don't
+have a very convenient interface in the form that sentry_sdk sends them to us in.
+They're dicts of dicts and tuples, and the structure changes depending on
+whether the event is an exception raised or an error logged. This class
+provides a more convenient interface to the interesting properties of events.
+"""
+
+from __future__ import unicode_literals
+
+
+class Event(object):
+    """A decorator for a sentry_sdk event."""
+
+    def __init__(self, event, hint):
+        self._event = event
+        self._hint = hint
+
+    @property
+    def event(self):
+        """The raw event dict as passed to us by sentry_sdk."""
+        return self._event
+
+    @property
+    def hint(self):
+        """The raw hint dict as passed to us by sentry_sdk."""
+        return self._hint
+
+    @property
+    def logger(self):
+        """
+        The name of the logger that logged the error message.
+
+        A string, or None of this isn't a logger event.
+        """
+        return self.event.get("logger")
```

Comment:
I've used properties for each of these (rather than simply assigning `self.logger = self.event.get("logger")` in the `__init__()` method above) because it enables `mock.create_autospec()` to detect them. It also nicely separates the different properties into different methods with docstrings, and scales more nicely. Makes them read-only too.